index.html

```diff
@@ -143,5 +143,36 @@ <h2>Skills</h2>
 		</div>
 	</section>
 
+	<header id="contact-header"></header>
+
+	<section id="contact">
+		<h1>Get started</h1>
+		<p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sit quidem a velit in tempore repellendus maiores eligendi inventore corporis, laborum architecto amet tenetur beatae quos quia numquam porro, rerum commodi.</p>
+		<form action="/" method="post">
```

Comment:
- [ ] Kindly use the action attribute to integrate with [formspree service](https://formspree.io/html).
      - `action` value should be similar to `https://formspree.io/f/{form_id}`